python/lsst/ts/ess/controller/device/rpi_serial_hat.py

```diff
@@ -37,31 +37,40 @@ class RpiSerialHat(common.device.BaseDevice):
 
     Parameters
     ----------
+    name : `str`
+        The name of the device.
     device_id : `str`
-        The hardware device ID to connect to, for instance /dev/ttyAMA1.
-    sensor : `common.sensor.BaseSensor`
+        The hardware device ID to connect to. This needs to be a physical ID
+        (e.g. /dev/ttyUSB0).
+    sensor : `BaseSensor`
         The sensor that produces the telemetry.
+    baud_rate : `int`
+        The baud rate of the sensor.
     callback_func : `Callable`
-        Callback function to receive instrument output.
+        Callback function to receive the telemetry.
+    log : `logging.Logger`
+        The logger to create a child logger for.
     """
 
     def __init__(
         self,
         name: str,
         device_id: str,
         sensor: common.sensor.BaseSensor,
+        baud_rate: int,
```

Comment:
Would a default value make sense? It's certainly not necessary, but you made a point that the baud rate was the same for most of your devices. A counter-argument is that I *think* that the way you construct these (except, perhaps, in unit tests) is always with a specified baud rate. In which case a default does little beyond adding a bit of documentation that could just be part of the doc string.

If you do change this, the argument will have to be last, since none of the others have defaults, and please consider changing VcpFtdi as well.